go-apps/meep-rnis/server/rnis_test.go

```diff
@@ -0,0 +1,2093 @@
+                                    "externalConfig": null,
+                                    "status": null,
+                                    "userChartLocation": null,
+                                    "userChartAlternateValues": null,
+                                    "userChartGroup": null,
+                                    "meta": null,
+                                    "userMeta": null,
+                                    "appLatency": null,
+                                    "appLatencyVariation": null,
+                                    "appPacketLoss": null,
+                                    "placementId": null
+                                 }
+                              ],
+                              "linkThroughput": 1000,
+                              "isExternal": null,
+                              "networkLocationsInRange": null,
+                              "meta": null,
+                              "userMeta": null,
+                              "linkLatency": null,
+                              "linkLatencyVariation": null,
+                              "linkPacketLoss": null
+                           }
+                        ],
+                        "terminalLinkPacketLoss": null,
+                        "meta": null,
+                        "userMeta": null,
+                        "cellularPoaConfig": null
+                     }
+                  ],
+                  "interFogLatency": null,
+                  "interFogLatencyVariation": null,
+                  "interFogThroughput": null,
+                  "interFogPacketLoss": null,
+                  "interEdgeLatency": null,
+                  "interEdgeLatencyVariation": null,
+                  "interEdgeThroughput": null,
+                  "interEdgePacketLoss": null,
+                  "edgeFogLatency": null,
+                  "edgeFogLatencyVariation": null,
+                  "edgeFogThroughput": null,
+                  "edgeFogPacketLoss": null,
+                  "meta": null,
+                  "userMeta": null
+               }
+            ],
+            "interZonePacketLoss": null,
+            "meta": null,
+            "userMeta": null
+         }
+      ],
+      "interDomainPacketLoss": null,
+      "meta": null,
+      "userMeta": null
+   },
+   "id": null,
+   "description": null,
+   "config": null
+}
+`
+const redisTestAddr = "localhost:30380"
+const influxTestAddr = "http://localhost:30986"
+
+func TestNotImplemented(t *testing.T) {
+	fmt.Println("--- ", t.Name())
+	log.MeepTextLogInit(t.Name())
+
+	fmt.Println("TestNotImplemented()")
+
+	//s1_bearer_info
+	_, err := sendRequest(http.MethodGet, "/queries/s1_bearer_info", nil, nil, nil, http.StatusNotImplemented, S1BearerInfoGET)
+	if err != nil {
+		t.Error("Failed to get expected response")
+	}
+	//rab_info
+	_, err = sendRequest(http.MethodGet, "/queries/rab_info", nil, nil, nil, http.StatusNotImplemented, RabInfoGET)
```

Comment:
Why not use the generated REST client package to send the requests as well?
Since you are using the package already to populate the HTTP request body in some cases, why not use it to send the requests as well?